frb_codegen/src/ir/field.rs

```diff
@@ -7,6 +7,7 @@ crate::ir! {
 pub struct IrField {
     pub ty: IrType,
     pub name: IrIdent,
+    pub wrapper_type: Option<String>,
```

Comment:
nit: May I know what is "wrapper_type" vs "ty", or can we infer the wrapper_type from the ty?

If we need to keep this field, would be great to add a bit of comments, explaining to readers this difference

Reply:
depends, It could maybe be a simple bool, I can give it a try